### PR TITLE
Brightest pixel location may occur at multiple position

### PR DESCRIPTION
**Pull request by prateekiiest**
@Cadair 

This corresponds to a wrong assumption in a gallery example - http://docs.sunpy.org/en/stable/generated/gallery/brightness_pixel_location.html#sphx-glr-generated-gallery-brightness-pixel-location-py

### CODE REVIEW

**Comment by prateekiiest** [comment]
Error due to import format for python 3, I guess

**Comment by prateekiiest** [comment]
https://github.com/sunpy/sunpy/issues/2323
Test falls  same error

**Comment by Cadair** [comment]
CI fails should be fixed by #2321 when merged.

File: examples/brightness_pixel_location.py [hunk 1]
```diff
@@ -20,10 +20,11 @@
...
 
 
-max_indices = np.unravel_index(aia.data.argmax(), aia.data.shape) * u.pixel
-
-hpc_max = aia.pixel_to_data(max_indices[1], max_indices[0])
-
+pixel_pos = np.argwhere(aia.data == aia.data.max())*u.pixel
+hpc_max = aia.pixel_to_data(pixel_pos[:,0], pixel_pos[:,1])
+    
+
```
Comment (Cadair) [inline examples/brightness_pixel_location.py]: too many line breaks, should only be one.

**Comment by Cadair** [comment]
This is broken:

figure_1-1

**Comment by Cadair** [comment]
@prateekiiest please make sure you run code locally before pushing it, this is a pretty obvious error if you look at the plot.

**Comment by prateekiiest** [comment]
Updated it

**Comment by nabobalis** [comment]
If doc build passes, I will merge this and backport.

**Pull request merged**